### PR TITLE
chore: add latency info

### LINKED ISSUE #5: Latency Sensor
Ping From Costa Rica in MS 

Later on we could add probes from different locations

### PR DESCRIPTION
**Pull request by adriexnet**
### What does this PR do?
- Add producer table
- Add worker to sync producers and rewards
- Add worker to sync producer bpjson
- Add worker to sync producer info and latency
- Change logic to display producers from the backend
- Add column "Ping from Costa Rica" for producer list
- Resolve #5

### How should this be tested?
- make run
- go to `http://localhost:3000`
- click "Block Producers" from side menu

### CODE REVIEW

**Comment by xavier506** [comment]
[CodeFactor](https://www.codefactor.io/repository/github/eoscostarica/eosio-monitor) found multiple issues last seen at 65042c8452ae66d3534b5df55aa8553d11e1f23e:

#### Replace `producer` with `(producer)`
[hapi\src\services\producer.service.js:70
](https://github.com/eoscostarica/eosio-monitor/blob/65042c8452ae66d3534b5df55aa8553d11e1f23e/hapi/src/services/producer.service.js#L70)[hapi\src\services\producer.service.js:139
](https://github.com/eoscostarica/eosio-monitor/blob/65042c8452ae66d3534b5df55aa8553d11e1f23e/hapi/src/services/producer.service.js#L139)[hapi\src\services\producer.service.js:230
](https://github.com/eoscostarica/eosio-monitor/blob/65042c8452ae66d3534b5df55aa8553d11e1f23e/hapi/src/services/producer.service.js#L230)[hapi\src\services\producer.service.js:177
](https://github.com/eoscostarica/eosio-monitor/blob/65042c8452ae66d3534b5df55aa8553d11e1f23e/hapi/src/services/producer.service.js#L177)
#### Replace `votes` with `(votes)`
[hapi\src\services\producer.service.js:123
](https://github.com/eoscostarica/eosio-monitor/blob/65042c8452ae66d3534b5df55aa8553d11e1f23e/hapi/src/services/producer.service.js#L123)
#### Replace `where` with `(where)`
[hapi\src\services\producer.service.js:39
](https://github.com/eoscostarica/eosio-monitor/blob/65042c8452ae66d3534b5df55aa8553d11e1f23e/hapi/src/services/producer.service.js#L39)
#### Replace `seconds` with `(seconds)`
[hapi\src\workers\producers.worker.js:4
](https://github.com/eoscostarica/eosio-monitor/blob/65042c8452ae66d3534b5df55aa8553d11e1f23e/hapi/src/workers/producers.worker.js#L4)
#### Replace `resolve` with `(resolve)`
[hapi\src\workers\producers.worker.js:5
](https://github.com/eoscostarica/eosio-monitor/blob/65042c8452ae66d3534b5df55aa8553d11e1f23e/hapi/src/workers/producers.worker.js#L5)

File: webapp/package.json [hunk 1]
```diff
@@ -24,19 +23,29 @@
     "not op_mini all"
   ],
   "dependencies": {
+    "@apollo/react-hooks": "^4.0.0",
     "@eoscostarica/eoscr-components": "~1.1.5",
```
Comment (xavier506) [inline webapp/package.json:27]: should we update this?

**Comment by adriexnet** [comment]
We have a problem with code factor because the linter with our configuration in local don't found any issue but code factor found issues with a rule that it's off in both .eslintrc files, I think that for now we can allow merge with problems related to arrow-parens rule and then take a look to the configuration about that rule and code factor

**Comment by xavier506** [comment]
> We have a problem with code factor because the linter with our configuration in local don't found any issue but code factor found issues with a rule that it's off in both .eslintrc files, I think that for now we can allow merge with problems related to arrow-parens rule and then take a look to the configuration about that rule and code factor

got it , we can ignore the messages then

File: webapp/package.json (hunk 1 above)
Comment (adriexnet) [reply webapp/package.json:27]: yes, you are right I will update it in my next PR

**Comment by adriexnet** [comment]
> Aweome work !
> 
> It brings up a couple of questions maybe we can talk about later
> 
> Can we enable websockets and subscriptions to hasura graphql engine?
> 
> How are historical metrics going to be persisted and shown?

I already using hasura subscriptions, for a live environment with need to add a few setting in the nginx proxy to can use the subscriptions in the same way that lifebank does 

The historical metrics it's not ready yet, actually its something that I want talk with you

**Pull request merged**